### DISCUSSION TITLE
Scanning internal sites

### DISCUSSION THREAD

**Discussion by Thato-Lesetla** [GoogleChrome/lighthouse#13561] [General] [answered]
Is it possible to scan sites that are internal to the company?

**Comment by adamraine** [comment, marked as answer]
Yes Lighthouse should work if your computer has access to the internal URL. If you need to authenticate you can check out our docs on the topic: https://github.com/GoogleChrome/lighthouse/blob/master/docs/recipes/auth/README.md.

**Reply by Thato-Lesetla** [reply]
Yes I see I'm not getting any issues when scanning the internal sites this is really impressive. Great work @adamraine
